gen/definitions/generic/hub_and_spoke_topology_policy_definition.yaml

```diff
@@ -47,6 +47,22 @@ attributes:
         mandatory: true
         description: Topology name
         example: Topology1
+      - model_name: equalPreference
```

Comment:
We need to add the `test_tags` attribute here, to ensure we are only testing this with 20.12 as it is not supported in 20.9.